### ISSUE TITLE
KeyboardNav doesn't allow keyHandlers to access links

### ISSUE THREAD

**Issue by slclarkdev** [AllenFang/react-bootstrap-table#1659]
When using KeyboardNav, I am unable to access any links in cells that are clickable via the mouse.  This is an accessibility issue.  

There doesn't seem to be a workaround since I can't attach a handler to the table cell.

<BootstrapTable ref='table' data={this.props.xmlTagList} striped keyBoardNav
                            containerClass='XMLTableClass'>
                        <TableHeaderColumn dataField='MW_XS_TAG_NAME' width='200px' cdataFormat={TagNameLinkFormatter.bind(this)} isKey dataSort={ true }>Tag Name</TableHeaderColumn>
</BootstrapTable>

function TagNameLinkFormatter(cell,row){
    let LinkString = `/schemaview/id/${row.MW_XS_ID}`
    return (
       <Link to={LinkString}>{cell}</Link>
    )
}

**Comment by AllenFang** [comment]
you mean the link is inactive when you click on links ?
